maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -51359,6 +51384,20 @@
 	icon_state = "sterile_green"
 	},
 /area/almayer/medical/containment)
+"mog" = (
+/obj/structure/safe,
```

Comment:
why not use the safe subtype that i made for CO?
/obj/structure/safe/co_office
it contain everything you are putting in...
it's probably been merge before you started your PR....
main purpose is to be able to change content without having to change map...

Reply:
What does it have?

Reply:
![image](https://github.com/cmss13-devs/cmss13/assets/117036822/37d2d653-5eb2-4e8a-af63-995977622cc7)

Reply:
https://github.com/cmss13-devs/cmss13/pull/4574/files

Reply:
so everything you put in it in your change and if you want to add more you can modify the subtype directly...
even made a subtype for the coin...